packages/cli/src/push/__tests__/websocket.push.test.ts

```diff
@@ -38,14 +40,14 @@ describe('WebSocketPush', () => {
 	});
 
 	it('can add a connection', () => {
-		webSocketPush.add(pushRef1, mockWebSocket1);
+		webSocketPush.add(pushRef1, userId, mockWebSocket1);
 
 		expect(mockWebSocket1.listenerCount('close')).toBe(1);
 		expect(mockWebSocket1.listenerCount('pong')).toBe(1);
 	});
 
 	it('closes a connection', () => {
-		webSocketPush.add(pushRef1, mockWebSocket1);
+		webSocketPush.add(pushRef1, userId, mockWebSocket1);
 
 		mockWebSocket1.emit('close');
 
```

Comment:
**suggestion:** Assert `expect(mockWebSocket1.listenerCount('message')).toBe(0)`

This requires to remove all listeners in the before each:

```ts
	beforeEach(() => {
		jest.resetAllMocks();
		mockWebSocket1.removeAllListeners();
		mockWebSocket2.removeAllListeners();
	});
```